apps/src/signUpFlow/LoginTypeSelection.tsx

```diff
@@ -1,3 +1,4 @@
+import {screen} from '@testing-library/react';
```

Comment:
Can this be used outside of a test?

Reply:
From a quick google search, it looks like it isn't a problem, but it's not best practice so I switched it to a document.getElementById instead!